codespell_lib/data/dictionary.txt

```diff
@@ -25216,7 +25241,7 @@ standartized->standardized
 standarts->standards
 standatd->standard
 standtard->standard
-standy->standby
+standy->standby, sandy, standee,
 stange->strange
 stanp->stamp
 staration->starvation
```

Comment:
I wonder if it's worth suggesting staration->saturation?

Reply:
Feels a bit of a stretch, it's a swap AND a missed letter. If you can find some examples where that's the intention I'll happily add it.